commands/util/pvp-rank.js

```diff
@@ -0,0 +1,161 @@
+const log = require('loglevel').getLogger('Counters'),

```

Comment:
This needs to be `RankCommand` instead of `Counters` so logging is correctly pointing to the correct location.

Reply:
Ok, thank you.